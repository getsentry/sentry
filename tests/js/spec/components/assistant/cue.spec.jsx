import React from 'react';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';
import AssistantCue from 'app/components/assistant/cue';

describe('AssistantCue', function() {
  it('renders', function() {
    let component = shallow(<AssistantCue cue="abc" onClick={() => 'test'} />);
    expect(toJson(component)).toMatchSnapshot();
  });
});
