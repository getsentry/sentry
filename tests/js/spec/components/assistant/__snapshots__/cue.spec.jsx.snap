// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AssistantCue renders 1`] = `
<div
  className="assistant-cue"
  onClick={[Function]}
/>
`;
